.husky/commit-msg

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-commitlint --edit ${1}
+npx commitlint --edit ${1}
```

Comment:
What's this for?

Reply:
dunno. i couldn't commit without it. i don't have commitlint installed globally i guess, nor do i want to. seems like this is the way to get husky to use the commitlint installed in devdeps.

could also mess with PATH in this script but `npx` seemed easier.